README.md

```diff
@@ -1,24 +1,597 @@
+    <img src="images/logo.png" alt="Logo" width="80" height="80">
+  </a>
 
-* Database creation
+<h3 align="center">Tavern Keeper</h3>
 
-* Database initialization
+  <p align="center">
+    INSERT-DETAILS-HERE
+    <br />
+    <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be">View Demo</a>
+    ·
+    <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/Tavern-Keeper-2308/tavern_keeper_be/issues">Request Feature</a>
+  </p>
+</div>
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+  1. [About The Project](#about-the-project)
+      - [Built With](#built-with)
+      - [Gems](#gems)
+  2. [Getting Started](#getting-started)
+      - [Installation](#installation)
+      - [Testing](#testing)
+  3. [Database](#database)
+  4. [Endpoints](#endpoints)
+      - [Query User](#q-user)
+      - [Query Encounter](#q-encounter)
+      - [Mutation New User](#m-new-user)
+      - [Mutation New Encounter](#m-new-encounter)
+  5. [Roadmap](#roadmap)
+  6. [Contributors](#contributors)
 
-* ...
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+<img src="images/landing.png" alt="landing" width="400" height="200">
+
+[![Tavern Keeper](https://img.shields.io/badge/Tavern_Keeper-Visit_Website-purple?style=for-the-badge)](INSERT-DEPLOY-LINK)
+
+### Built With
+* [![Ruby on Rails][Rails-shield]][Rails-url]
+* [![GraphQL][GraphQL-shield]][GraphQL-url]
+* [![PostgreSQL][PostgreSQL-shield]][PostgreSQL-url]
+* [![HTML][HTML-shield]][HTML-url]
+* [![JavaScript][JavaScript-shield]][JavaScript-url]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Gems
+
+#### Deployment
+* [![faraday][gem-faraday]][gem-faraday-url]
+* [![jsonapi-serializer][gem-jsonapi-serializer]][gem-jsonapi-serializer-url]
+
+#### Testing
+* [![debug][gem-debug]][gem-debug-url]
+* [![rspec-rails][gem-rspec-rails]][gem-rspec-rails-url]
+* [![simplecov][gem-simplecov]][gem-simplecov-url]
+* [![factory_bot_rails][gem-factory_bot_rails]][gem-factory_bot_rails-url]
+* [![faker][gem-faker]][gem-faker-url]
+* [![pry][gem-pry]][gem-pry-url]
+* [![shoulda-matchers][gem-shoulda-matchers]][gem-shoulda-matchers-url]
+* [![capybara][gem-capybara]][gem-capybara-url]
+* [![webmock][gem-webmock]][gem-webmock-url]
+* [![vcr][gem-vcr]][gem-vcr-url]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Database 
+
+<img src="images/schema.png"  width="300" height="800">
+
+### Installation
+
+1. Get an API Key at [INSERT-API-LOCATION-PATH-HERE](INSERT-API-LOCATION-PATH-HERE)
```

Comment:
I think we should get rid of the references to API keys in this document for now. We can always add this later and keyed authentication was not part of our MVP.

Reply:
I have removed references to using an API key, I actually did not realize the DnD one was keyless. Thanks!